tests/test_memdumps.py

```diff
@@ -84,31 +84,10 @@ def _runManticore(self, dumpname):
 
         self.assertGreaterEqual(actual, expected)
 
-    @unittest.skip('Windows temporarily unmaintained')
```

Comment:
It looks like the rest of this file isn't used anymore with these tests deleted.